lib/src/base-styles/_nx-tables.scss

```diff
@@ -365,29 +365,89 @@
   }
 }
 
-.nx-cell__sort-btn {
-  height: nx-table-variables.$table-cell-header-height;
+.nx-cell {
+  .nx-cell__sort-btn {
+    height: nx-table-variables.$table-cell-header-height;
+    position: relative;
+  
+    > * {
+      position: relative;
+    }
+  
+    &::before {
+      box-sizing: border-box;
+      border-radius: 4px;
+      content: '';
+      // offset by 1px on top and 2px on the bottom
+      height: calc(100% - 3px);
+      position: absolute;
+      left: 0;
+      top: 1px;
+      width: calc(100% - 1px);
```

Comment:
I think this is too wide: 
![image](https://github.com/sonatype/sonatype-react-shared-components/assets/3630091/172bf818-917a-4433-b89c-22589ad2c2d1)

Reply:
> I think this is too wide:

Fixed here: [f488f71](https://github.com/sonatype/sonatype-react-shared-components/pull/1156/commits/f488f71b61f1f4d8dad07cb12bc5981bbeead1eb)